freedomofpress/securedrop
# Add globe icon in the locale dropdown for better understanding

As pointed out [here](https://forum.securedrop.club/t/language-menu-is-confusing-when-the-user-does-not-read-the-current-language/407), language menu is confusing when the user does not read the current language. Adding a globe icon helps understand that the dropdown is about locales for people who don't read the current language.

## Status
Ready for review

## Description of Changes
Added globe icon before(left of) the locale name and moved the trigger arrow after(right of) the locale name.
![screen shot 2018-03-08 at 1 40 08 pm](https://user-images.githubusercontent.com/9530293/37140745-d364211c-22d8-11e8-9043-5e5db56da0c0.png)

## Testing

How should the reviewer test this PR?
I did the following steps for testing:
1. Add locales in `SUPPORTED_LOCALES` list in the `securedrop/config.py` file
2. run `./i18n_tool.py translate-messages`
3. run `./manage.py run`
4. Open source interface and journalist interface to verify a globe icon appears in the language dropdown.

## Checklist

### If you made changes to the app code:
